src/gfakluge.cpp

```diff
@@ -1056,6 +1056,11 @@ namespace gfak{
             // for (auto f : seq_to_fragments[s.first]){
             //     ret << f.second.to_string() << endl;
             // }
+	    if (this->version < 2.0) { 
+            	for (auto e : seq_to_link[s.first]) { 
+            	ret << e.to_string() << endl; 
+             } 
```

Comment:
The design of the output function *should* mean that when gfa_2_ize() gets called seq_to_edges and seq_to_link contain the same information; adding this output should cause edges/links to be output twice. If this isn't the case I'll have to fix the gfa_2_ize function.

The reason for this change is I'd eventually like to deprecate the link_elem and containment_elem structs.

Reply:
Aaaand I see why this is a problem now - I never call gfa_2_ize() or gfa_1_ize() to homogenize the representation.

Reply:
Wasn't really sure if the if-clause was really needed, I didn't had time to look at the to_string_2 of the sequence struct so I added it just to be sure. This was just a quick fix, haven't look at all the code to be sure that this is "the right way". But thanks for incorporating it!